about.html

```diff
@@ -16,7 +16,7 @@
   The Jupyter <a href="https://jupyter.org/governance/software_steering_council.html">Software Steering Council</a> (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn’t explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
 
 contributors_copy: |
-  Project Jupyter <a href="https://jupyter.org/governance/distinguished_contributors.html">Distinguished Contributors</a> are recognized for their substantial contributions to Jupyter itself in both quality and quantity over at least two years. Contributions may include code, code review, infrastructure work, mailing list and chat participation, community help/building, education and outreach, fundraising, branding, marketing, inclusion and diversity, UX design and research, etc. Up to 10 new Distinguished Contributors are selected each year by the cumulative body of Distinguished Contributors.
+  Project Jupyter <a href="https://jupyter.org/governance/distinguished_contributors.html">Distinguished Contributors</a> are a group of Jupyter community members that have gone above-and-beyond in their support of the project over the years, making substantial and sustained contributions in any area of activity (software development, governance, community engagement, events, etc.). The Jupyter community confers membership in this group as a way of recognizing their effort and saying "thank you."
```

Comment:
```suggestion
  Project Jupyter <a href="https://jupyter.org/governance/distinguished_contributors.html">Distinguished Contributors</a> are a group of Jupyter community members that have gone above and beyond in their support of the project over the years, making substantial and sustained contributions in any area of activity, such as software development, governance, community engagement, and events. The Jupyter community confers membership in this group as a way of recognizing and thanking members for their effort.
```

Minor copy edit (should probably be made in the linked source material as well)

Reply:
Please let's make the edit in the source text first because the whole point of this PR is to rationalize the two documents.